<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCLHorizontalSlits</class>
 <widget class="QWidget" name="VCLHorizontalSlits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="minimumSize">
      <size>
       <width>224</width>
       <height>175</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QESubstitutedLabel" name="titleLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="slitFrame" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 200);</string>
        </property>
        <widget class="QWidget" name="widgetTopRight" native="true">
         <property name="geometry">
          <rect>
           <x>328</x>
           <y>154</y>
           <width>144</width>
           <height>122</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 210, 220);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>13</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget { background-color: rgba(0, 0, 0, 0); color: rgb(130, 4, 0); }</string>
            </property>
            <property name="text">
             <string>H Centre</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QELabel" name="hcMtrPosnMon">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>17</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="forceSign">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QELineEdit" name="hcMtrPosnSet">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>22</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>18</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QEPushButton" name="hcMtrTweakRev">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../dblslits/vcl_dblslits.qrc">
                 <normaloff>:/vcl/dblslits/vcl_ds_left.png</normaloff>:/vcl/dblslits/vcl_ds_left.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QELineEdit" name="hcMtrTweakVal">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QEPushButton" name="hcMtrTweakFwd">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../dblslits/vcl_dblslits.qrc">
                 <normaloff>:/vcl/dblslits/vcl_ds_right.png</normaloff>:/vcl/dblslits/vcl_ds_right.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>240</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QESimpleShape" name="outerBlade">
         <property name="geometry">
          <rect>
           <x>311</x>
           <y>20</y>
           <width>172</width>
           <height>116</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="edgeWidth">
          <number>3</number>
         </property>
         <property name="modulus">
          <number>2</number>
         </property>
         <property name="format" stdset="0">
          <enum>QSimpleShape::PvText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="colour0" stdset="0">
          <color alpha="80">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="colour1" stdset="0">
          <color alpha="80">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="displayAlarmStateOption" stdset="0">
          <enum>QESimpleShape::Never</enum>
         </property>
        </widget>
        <widget class="QWidget" name="widgetTopLeft" native="true">
         <property name="geometry">
          <rect>
           <x>58</x>
           <y>154</y>
           <width>128</width>
           <height>122</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 210, 220);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>13</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>13</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget { background-color: rgba(0, 0, 0, 0); color: rgb(130, 4, 0); }</string>
            </property>
            <property name="text">
             <string>H Size</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QELabel" name="hsMtrPosnMon">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>17</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QELineEdit" name="hsMtrPosnSet">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>22</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_1" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>18</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QEPushButton" name="hsMtrTweakRev">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../dblslits/vcl_dblslits.qrc">
                 <normaloff>:/vcl/dblslits/vcl_ds_neg.png</normaloff>:/vcl/dblslits/vcl_ds_neg.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QELineEdit" name="hsMtrTweakVal">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QEPushButton" name="hsMtrTweakFwd">
               <property name="minimumSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../dblslits/vcl_dblslits.qrc">
                 <normaloff>:/vcl/dblslits/vcl_ds_pos.png</normaloff>:/vcl/dblslits/vcl_ds_pos.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>240</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QESimpleShape" name="innerBlade">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>172</width>
           <height>116</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="edgeWidth">
          <number>3</number>
         </property>
         <property name="modulus">
          <number>2</number>
         </property>
         <property name="format" stdset="0">
          <enum>QSimpleShape::PvText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="colour0" stdset="0">
          <color alpha="80">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="colour1" stdset="0">
          <color alpha="80">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="displayAlarmStateOption" stdset="0">
          <enum>QESimpleShape::Never</enum>
         </property>
        </widget>
        <widget class="QLabel" name="outerLabel">
         <property name="geometry">
          <rect>
           <x>409</x>
           <y>24</y>
           <width>64</width>
           <height>15</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>15</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Outboard</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>4</number>
         </property>
        </widget>
        <widget class="QLabel" name="innerLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>24</y>
           <width>64</width>
           <height>15</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>15</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Inboard</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>4</number>
         </property>
        </widget>
        <zorder>widgetTopRight</zorder>
        <zorder>outerBlade</zorder>
        <zorder>widgetTopLeft</zorder>
        <zorder>innerBlade</zorder>
        <zorder>innerLabel</zorder>
        <zorder>outerLabel</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QESubstitutedLabel</class>
   <extends>QLabel</extends>
   <header>QESubstitutedLabel.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../dblslits/vcl_dblslits.qrc"/>
 </resources>
 <connections/>
</ui>
